router.js

```diff
@@ -45,9 +45,9 @@ router.get("/load", async (ctx) => {
   d = d.replace("dataPlaneUrl", dataPlaneUrl);
   d = d.replace("configBackendUrl", configUrl);
 
-  deviceModeInit = deviceModeInit.replaceAll("dataplaneUrl", dataPlaneUrl);
-  deviceModeInit = deviceModeInit.replaceAll("writeKey_placeHolder", writeKey);
-  deviceModeInit = deviceModeInit.replaceAll("configUrl_placeholder", configUrl);
+  deviceModeInit = deviceModeInit.replace(/dataplaneUrl_placeHolder/g, dataPlaneUrl);
+  deviceModeInit = deviceModeInit.replace(/writeKey_placeHolder/g, writeKey);
+  deviceModeInit = deviceModeInit.replace(/configUrl_placeholder/g, configUrl);
```

Comment:
I think in device mode init the config url is stored in a variable called webhookURL can we change that

Reply:
Why do we need to change that it is used to fetch the` /sourceConfig`